Allow polling on metrics dashboard to stop polling.

anwar6953 opened the pull request:
Previously, you could not suspend the polling frequency except by navigating to a different page or applying a Time Range.
This allows you to stop the polling.

elfenheart commented on cdap-ui/app/features/dashboard/templates/userdashboard.html:
```diff
@@ -72,9 +72,12 @@ <h3 class="text-center"> No widgets on this board! <a ng-click="addWidget()"> Ad
+          <button ng-if="!liveDashboard" class="btn btn-success" ng-click="updateWithFrequency()">
             Apply!
           </button>
+          <button ng-if="liveDashboard" class="btn btn-success" ng-click="stopPolling()">
```
I think this should be btn-danger

anwar6953 replied:
updated.

anwar6953 commented:
Changed Time Range button to `Make Request`.
Changed Frequency button to `Start Polling` and `Stop Polling`
---------------------
![image](http://g.recordit.co/VVx5L07u0p.gif)

elfenheart commented:
:+1: LGTM
